test/UnitTest/Extensions/HttpClientExtensionTests.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Extensions;
+using Altinn.Platform.Storage.Tests.Stubs;
+
+using Xunit;
+
+namespace Altinn.Platform.Storage.Tests.Extensions
+{
+    public class HttpClientExtensionTests
+    {
+        private readonly HttpClient _httpClient;
+        private HttpRequestMessage _httpRequest;
+
+        public HttpClientExtensionTests()
+        {
+            var httpMessageHandler = new DelegatingHandlerStub(async (request, token) =>
+            {
+                _httpRequest = request;
+                await Task.Yield();
+                return new HttpResponseMessage(HttpStatusCode.OK);
```

Comment:
This feels safer. It's something we've already used quite a bit:
```suggestion
                return await Task.FromResult(new HttpResponseMessage(HttpStatusCode.OK));
```